### ISSUE TITLE
Error in Next.js (fixed but need assistance merging)

### ISSUE THREAD

**Issue by Kiesco08** [mobily/stacks#21]
When used in Next.js, this error is thrown:

```
./node_modules/@mobily/stacks/dist/index.mjs Attempted import error: 'react-native-web/dist/index' does not contain a default export (imported as 'ReactNative').
```

I was able to fix it by going into `node_modules/@mobily/stacks/dist/index.mjs` and updating every instance of `import ReactNative` with `import * as ReactNative`.

I would create a PR, but I'm not familiar with ReScript. Some help would be appreciated.

**Comment by Kiesco08** [comment]
Hey @mobily, I have a basic config from https://github.com/vercel/next.js/tree/canary/examples/with-expo-typescript

`next.config.js`:
```
const { withExpo } = require('@expo/next-adapter')
const withPlugins = require('next-compose-plugins')
const withTM = require('next-transpile-modules')(['@mobily/stacks'])

module.exports = withPlugins([withTM, [withExpo, { projectRoot: __dirname }]], {
  /* next.config.js code */
})
```

**Comment by mobily** [comment]
@Kiesco08 fixed in `v1.1.5`, you can find more details about the RNW config in the docs: https://mobily.github.io/stacks/docs/react-native-web
